apps/qubit/modules/informationobject/actions/reportsAction.class.php

```diff
@@ -50,11 +55,17 @@ protected function addField($name)
           $choices[$this->context->routing->generate(null, array($this->resource, 'module' => 'informationobject', 'action' => 'boxLabelCsv'))] = $this->context->i18n->__('Box label CSV');
         }
 
-        $this->form->setDefault($name, $this->context->routing->generate(null, array($this->resource, 'module' => 'informationobject', 'action' => 'fileList')));
-        $this->form->setValidator($name, new sfValidatorChoice(array('choices' => array_keys($choices))));
-        $this->form->setWidget($name, new sfWidgetFormChoice(array(
-          'expanded' => true,
-          'choices' => $choices)));
+
+        $this->reportsAvailable = !empty($choices);
+
+        if ($this->reportsAvailable) {
+          $available_routes = array_keys($choices);
+          $this->form->setDefault($name, $available_routes[0]);
+          $this->form->setValidator($name, new sfValidatorChoice(array('choices' => array_keys($choices))));
```

Comment:
Also minor, could use $available_routes here:

$this->form->setValidator($name, new sfValidatorChoice(array('choices' => $available_routes)));